ctapipe/io/astropy_helpers.py

```diff
@@ -88,6 +97,101 @@ def read_table(h5file, path, start=None, stop=None, step=None, condition=None) -
         return astropy_table
 
 
+def write_table(table, h5file, path, append=False, mode="a", filters=DEFAULT_FILTERS):
+    """Write a table to an HDF5 file
+
+    This writes a table in the ctapipe format into ``h5file``.
+
+                attrs.update(transform.get_meta(name))
+
+    Parameters
+    ----------
+    table: astropy.table.Table
+        The table to be written.
+    h5file: Union[str, Path, tables.file.File]
+        input filename or PyTables file handle. If a PyTables file handle,
+        must be opened writable.
+    path: str
+        dataset path inside the ``h5file``
+    append: bool
+        Wether to try to append to or replace an existing table
+    mode: str
+        If given a path for ``h5file``, it will be opened in this mode.
+        See the docs of ``tables.open_file``.
+    """
+    copied = False
+
+    with ExitStack() as stack:
+
+        if isinstance(h5file, (str, Path)):
+            h5file = stack.enter_context(tables.open_file(h5file, mode=mode))
+        elif isinstance(h5file, tables.file.File):
+            pass
+        else:
+            raise ValueError(
+                f"expected a string, Path, or PyTables "
+                f"filehandle for argument 'h5file', got {h5file}"
+            )
+
+        attrs = {}
+        for colname, column in table.columns.items():
+            if hasattr(column, "description") and column.description is not None:
+                attrs[f"{colname}_DESC"] = column.description
+
+            if isinstance(column, Time):
+                transform = TimeColumnTransform(scale="tai", format="mjd")
+                attrs.update(transform.get_meta(colname))
+
+                if copied is False:
+                    table = table.copy()
+                    copied = True
+
+                table[colname] = transform(column)
+
+            # convert unicode to fixed-lenth bytes
+            # TODO use variable length strings as soon as tables supports them
```

Comment:
I suppose thats this issue?
https://github.com/PyTables/PyTables/issues/48
Maybe add the link for future reference?